src/frameworks/gatsby.json

```diff
@@ -23,6 +23,11 @@
     "AWS_LAMBDA_JS_RUNTIME": "nodejs14.x",
     "NODE_VERSION": "14"
   },
+  "logo": {
+    "default": "default.svg",
+    "light": "light.svg",
+    "dark": "dark.svg"
```

Comment:
todo before merging: these need to be updated to the URL where these assets are hosted. Currently there's a [test site](https://ep-framework-info.netlify.app/logos/gatsby/default.svg) deployed that's working, but we'll need to delete the old `framework-info` site so that we can update the URL to `https://framework-info.netlify.app/logos/gatsby/default.svg

Reply:
Since the change set now uses the `DEPLOY_PRIME_URL` rather than the hard coded path, we don't need to worry about this